plenum/test/view_change/test_no_propagate_request_on_different_last_ordered_before_vc.py

```diff
@@ -47,19 +47,19 @@ def test_no_propagate_request_on_different_last_ordered_on_backup_before_vc(loop
     primary = getPrimaryReplica(txnPoolNodeSet, slow_instance).node
     non_primaries = [n for n in txnPoolNodeSet if n is not primary]
 
-    check_last_ordered(non_primaries,
-                       slow_instance,
-                       (old_view_no, old_last_ordered[1] + 1))
+    looper.run(eventually(check_last_ordered, non_primaries,
+                          slow_instance,
+                          (old_view_no + 1, 1)))
 
     # Backup primary replica must not advance last_ordered_3pc
     # up to the master's value
-    check_last_ordered([primary],
-                       slow_instance,
-                       (old_view_no, old_last_ordered[1]))
+    looper.run(eventually(check_last_ordered, [primary],
```

Comment:
This doesn't match the comment that backup primary should not change last_ordered

Reply:
fixed